Download.Mvc/Client/components/SearchEntries.tsx

```diff
@@ -0,0 +1,18 @@
+﻿import * as React from 'react';
+import { SearchEntry } from './SearchEntry';
+
+export class SearchEntries extends React.Component<any, any> {
+
+    render() {
+        let entryList = "";
+        if (this.props.items) {
+            entryList = this.props.items.map((entry, index) =>
+                <SearchEntry key={index} entry={entry} />);
+        }
+        return (
+            <div>
+                {entryList}
+            </div>
+        );
+    }
```

Comment:
@laholstege very cool map here!  One thing that could clean this up though is to just do `if (!this.props.items) return null` and then create your entry list w/ the map and return the div, instead of having to default to an empty string.  Also if you default go for null instead of ""